packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -86,12 +87,124 @@ const [phpLoaderModule, wpLoaderModule] = await Promise.all([
 ]);
 monitor.setModules([phpLoaderModule, wpLoaderModule]);
 php.initializeRuntime(
-	await loadPHPRuntime(phpLoaderModule, monitor.getEmscriptenArgs(), [
-		wpLoaderModule,
-	])
+  await loadPHPRuntime(phpLoaderModule, {
+    ...monitor.getEmscriptenArgs(), 'print': function (text) {
+      console.log('stdout: ' + text)
+    },
+    'printErr': function (text) {
+      console.log('stderr: ' + text)
+    }
+  }, [
+    wpLoaderModule,
+  ])
 );
 patchWordPress(php, scopedSiteUrl);
 
+
+php.writeFile(
+  '/wordpress/tests/phpunit/includes/bootstrap.php',
+  php.readFileAsText('/wordpress/tests/phpunit/includes/bootstrap.php').replace("'gd',", '')
```

Comment:
WP core unit tests don't run if the `gd` extension is missing:

https://github.com/WordPress/wordpress-develop/blob/ad2405c10e4d6311464ff40d75435ad95bb8a844/tests/phpunit/includes/bootstrap.php#L166-L187

This monkey-patches the file to work around this.

Reply:
I once came across this too! I'd move this to Dockerfile, but the idea is sound – at least until this project ships GD as a runtime dependency.